### PR TITLE
Makes Glowing Goo Not PERMANENTLY radioactive

### PR DESCRIPTION
**Pull request by VaelophisNyx**
# Document the changes in your pull request

reasons you shouldn't blindly merge PRs:

just makes radiation from glowing goo go away with the light too.

# Changelog

:cl:  
bugfix: glowing goo from spilled radium or uranium is no longer PERMANENTLY radioactive
/:cl:

### CODE REVIEW

File: code/game/objects/effects/decals/cleanable/misc.dm [hunk 1]
```diff
@@ -89,10 +89,12 @@
 	addtimer(CALLBACK(src, .proc/Decay), 24 SECONDS)
 
 /obj/effect/decal/cleanable/greenglow/proc/Decay()
+	var/datum/component/radioactive/R = src.GetComponent(/datum/component/radioactive)
 	name = "dried goo"
 	light_power = 0
 	light_range = 0
 	update_light()
+	R.RemoveComponent()
```
Comment (Bibby0110) [inline code/game/objects/effects/decals/cleanable/misc.dm]: Move this up below where you fetch the component, also check if an R was actually returned

File: code/game/objects/effects/decals/cleanable/misc.dm [hunk 2]
```diff
@@ -89,10 +89,12 @@
 	addtimer(CALLBACK(src, .proc/Decay), 24 SECONDS)
 
 /obj/effect/decal/cleanable/greenglow/proc/Decay()
+	var/datum/component/radioactive/R = src.GetComponent(/datum/component/radioactive)
```
Comment (Bibby0110) [inline code/game/objects/effects/decals/cleanable/misc.dm]:
```suggestion
	var/datum/component/radioactive/R = GetComponent(/datum/component/radioactive)
```

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/decals/cleanable/misc.dm (hunk 2 above)
Comment (VaelophisNyx) [reply code/game/objects/effects/decals/cleanable/misc.dm]: note that src. was required for this to work here. Tried it without and it didn't want to work

**Comment by ToasterBiome** [comment]
im waking up to ash and dust

**Comment by Hopekz** [comment]
Blind merge is the yogdev way

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**